E2E/pageObjects/game.page.ts

```diff
@@ -124,15 +128,50 @@ export = {
     I.switchToPreviousTab();
     I.seeElement(locator.chessBoard.board);
   }, 
+
   skipMove: () => {
     I.click(locator.buttons.skip);
     I.seeElement(locator.buttons.skipButtonHighlighted);
   },
+
   voteAndCheckThatVoteIsVisible:(chessPiece: ChessPiece, tile: string, value:string) => {
     game.vote(chessPiece, tile);
     game.voteIsVisible(chessPiece, tile, value);
   },
 
+  getExpectedPlayerSP:(averageScore) => {
+    const sp = [1,2,3,5,8,13];
+    const averageSPList = [1.5, 2.5, 4, 6.5, 10.5];
+    let expectedSP = 1;
+    for(let i = 0; averageSPList[i] <= averageScore; i++){
+      expectedSP = sp[i + 1];
+    };
+    return expectedSP;
+  },
+
+  calculateAverage:(piece: string, letter: string, number: number) => {
+    let piecesScore = {pawn:1, knight:2, bishop:3, rook:5, king:8, queen:13};
```

Comment:
We have chess pieces names in test_data/ChessPieces.ts file, that can be reused, or maybe even numbers from ChessPieceValue?